Docs/source/glossary.rst

```diff
@@ -22,12 +22,14 @@ Abbreviations
 * **CI:** continuous integration, automated tests that we perform before a proposed code-change is accepted; see PR
 * **CPU:** `central processing unit <https://en.wikipedia.org/wiki/Central_processing_unit>`__, we usual mean a socket or generally the host-side of a computer (compared to the accelerator, e.g. GPU)
 * **DOE:** `The United States Department of Energy <https://en.wikipedia.org/wiki/United_States_Department_of_Energy>`__, the largest sponsor of national laboratory research in the United States of America
+* **DSMC:** Direct Simulation Monte Carlo, a method to capture collisions between kinetic particles
```

Comment:
```suggestion
* **DSMC:** `Direct Simulation Monte Carlo <https://en.wikipedia.org/wiki/Direct_simulation_Monte_Carlo>`__, a method to capture collisions between kinetic particles
```
Should we add a link to the corresponding wikipedia page?

Reply:
Sounds good to me, thanks for the suggestion!